ir/instr.cpp

```diff
@@ -1674,6 +1674,50 @@ unique_ptr<Instr> Malloc::dup(const string &suffix) const {
   return make_unique<Malloc>(getType(), getName() + suffix, *size, isNonNull);
 }
 
+vector<Value*> Calloc::operands() const {
+  return { num, size };
+}
+
+void Calloc::rauw(const Value &what, Value &with) {
+  RAUW(num);
+  RAUW(size);
+}
+
+void Calloc::print(std::ostream &os) const {
+  os << getName() << " = calloc " << *num << ", " << *size;
+}
+
+StateValue Calloc::toSMT(State &s) const {
+  auto &[nm, np_num] = s[*num];
+  auto &[sz, np_sz] = s[*size];
+  s.addUB(np_num && np_sz);
```

Comment:
I feel that yielding UIB is too strong. Just realized that malloc does the same, but that's not well justified. Malloc won't crash or anything if poison is passed as input.
I think we should just return poison in this case.

Reply:
I'll fix malloc.

Reply:
I think this is related with whether calloc can be optimized to malloc + memset in LLVM. If only a portion of allocated bytes of calloc() is read, it is benefial to do memset on only those bytes. memset(src, 0, poison) is currently defined as UB.